homeassistant/components/airzone_cloud/sensor.py

```diff
@@ -98,6 +100,19 @@
         native_unit_of_measurement=PERCENTAGE,
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    SensorEntityDescription(
+        device_class=SensorDeviceClass.BATTERY,
+        key=AZD_THERMOSTAT_BATTERY,
+        native_unit_of_measurement=PERCENTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    SensorEntityDescription(
+        icon="mdi:signal",
+        key=AZD_THERMOSTAT_COVERAGE,
+        name="Signal Percentage",
```

Comment:
2 things
1. Translations (both entity name and icon translations) (i think this got migrated to had entity name so that would work)
2. Should this be disabled by default?

Reply:
Solved by:
1. ef1275f87371ecb765b3c8a9cb4a0c3b62ca77ae
2. f96363ae41395a9bd501a6d3284f34d62d46ad37